src/main/java/uk/gov/hmcts/reform/coh/schedule/notifiers/BasicJsonNotificationForwarder.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.coh.schedule.notifiers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.coh.domain.SessionEventForwardingRegister;
+
+import java.io.IOException;
+
+@Component
+@Qualifier("BasicJsonNotificationForwarder")
+public class BasicJsonNotificationForwarder implements NotificationForwarder<NotificationRequest> {
+
+    private static final Logger log = LoggerFactory.getLogger(BasicJsonNotificationForwarder.class);
+
+    private static final String PLACEHOLDER_HOST = "${base-urls.test-url}";
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    private static HttpHeaders URL_ENCODED_HEADER;
+
+    static {
+        URL_ENCODED_HEADER = new HttpHeaders();
+        URL_ENCODED_HEADER.add("Content-Type", "application/json");
+    }
+
```

Comment:
This is probably not the right way to fix the placeholder host name. I think the way is probably to have a post processing script to fix the endpoint in the DB.  It'll have to do for now.